### ISSUE TITLE
Unable to install Virtualbox plugin with Vagrant 1.6.2

### ISSUE THREAD

**Issue by bmf** [lynnaloo/node-vagrant#1]
**TL;DR** Got fed up and installed Vagrant 1.5.0. Problem solved (ignored).

I attempted to install Vagrant 1.6.2 and the Virtualbox plugin following the instructions that were provided in the Readme. When I tried the plugin, I received the following error: 

_Bundler, the underlying system Vagrant uses to install plugins,_
_reported an error. The error is shown below. These errors are usually_
_caused by misconfigured plugin installations or transient network_
_issues. The error from Bundler is:_

_An error occurred while installing nokogiri (1.6.2), and Bundler cannot continue._
_Make sure that `gem install nokogiri -v '1.6.2'` succeeds before bundling._

After some research I came across people reporting similar issues here: https://github.com/mitchellh/vagrant/issues/3769.

You're welcome to read through them, but I walked away with the following information:  A) Some people have the same issue. B) Some people have fixed it. C) Their fixes probably won't work on your machine.

After trying: `add s.add_dependency(%q<nokogiri>, ["< 1.6.2"])` to `~/Applications/Vagrant/embedded/gems/specifications/vagrant-1.6.2.gemspec` and reinstalling ruby the issue still did not resolve itself. I finally gave up and installed vagrant 1.5.0 which seems to work.
